### ISSUE TITLE
Error Checking for Minor Updates

### ISSUE THREAD

**Issue by rgilsimoes** [ionic-team/vscode-extension#126]
Getting this error when Checking for Minor Updates:
```
[Ionic] Checking for minor updates for 56 dependencies
[error] {"type":"warning","data":"package.json: No license field"}
{"type":"error","data":"No lockfile in this directory. Run `yarn install` to generate one."}

[error] {"type":"warning","data":"package.json: No license field"}
{"type":"error","data":"No lockfile in this directory. Run `yarn install` to generate one."}
```

Ionic VSCode Extension Version v1.45.5
VSCode Info
Version: 1.80.1 (user setup)
Commit: 74f6148eb9ea00507ec113ec51c489d6ffb4b771
Date: 2023-07-12T17:22:07.651Z
Electron: 22.3.14
ElectronBuildId: 21893604
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22621

**Comment by damiant** [comment]
Do you have a project that you can share? (This would be hard to reproduce). You can send a link to a zip to [redacted]

**Comment by rgilsimoes** [comment]
Hi Damian,

Thank you for the reply and sorry for the late response.
Here is the project directory:

https://drive.google.com/file/d/1L0svF4YT_kqGjQRhPF3Gk070UW1zJa9D/view?usp=sharing


Thanks,

Ricardo

[@] : ***@***.***
[m] : Rua Fialho de Almeida
         Lote 23, 2W
         Pampilhosa, 3050-413 Portugal
[w3]: http://www.linkedin.com/in/rsimoes


Damian ***@***.***> escreveu no dia sábado, 15/07/2023 à(s)
19:15:

**Comment by dtarnawsky** [comment]
Thanks for the project. This issue should now be fixed in 1.48

The problem was that in yarn 1.x projects the command `yarn outdated --json` returns invalid json. The extension now works around this bug. Not only will you be able to check for minor updates, you'll also see a lot more information when you expand packages/plugins.

**Comment by rgilsimoes** [comment]
It works after upgrade to 1.48
